src/css/custom.css

```diff
@@ -475,6 +475,10 @@ aside.theme-doc-sidebar-container {
   background-image: url("data:image/svg+xml,%3Csvg width='24' height='24' fill='none' viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M8.293 4.293a1 1 0 0 0 0 1.414L14.586 12l-6.293 6.293a1 1 0 1 0 1.414 1.414l7-7a1 1 0 0 0 0-1.414l-7-7a1 1 0 0 0-1.414 0Z' fill='%23181818'/%3E%3C/svg%3E");
 }
 
+.theme-doc-sidebar-container:hover div[title='Expand sidebar'] {
+  position: absolute;
+}
+
```

Comment:
Fix: When the sidebar menu is collapsed, the button for expansion will not be displayed, this style will make it display normally.

- before
<img width="309" alt="image" src="https://github.com/FedML-AI/doc.fedml.ai/assets/19882767/61b7a058-8a01-4955-ab64-79743f1c9901">

- after
<img width="256" alt="image" src="https://github.com/FedML-AI/doc.fedml.ai/assets/19882767/6bccb674-a38f-4713-bf0e-66b82670159f">